Tests/Admin/AdminTest.php

```diff
@@ -1524,18 +1524,6 @@ public function testGetSubject()
         $admin->setModelManager($modelManager);
 
         $admin->setRequest(new Request(array('id' => 'azerty')));
```

Comment:
@peter-gribanov : could you change "azerty" to something that looks like what happens in real life when using a composite key, and add a comment `// composite keys are supported` ?